FinSearchUnified/Tests/Helper/StaticHelperTest.php

```diff
@@ -55,6 +54,125 @@ protected function tearDown()
         Shopware()->Container()->load('session');
         Shopware()->Container()->reset('config');
         Shopware()->Container()->load('config');
+        Shopware()->Container()->reset('fin_search_unified.config_loader');
+        Shopware()->Container()->load('fin_search_unified.config_loader');
+    }
+
+    /**
+     * Data provider for checking isFindologicActive behavior
+     *
+     * @return array
+     */
+    public function isFindologicActiveDataprovider()
+    {
+        return [
+            'Plugin is deactivated' => [
+                'ActivateFindologic' => false,
```

Comment:
camelCase your data provider keys and make sure they have the same name as the method params that use these data providers.